backend/src/packages/chaiNNer_standard/image/video_frames/save_video.py

```diff
@@ -98,6 +98,62 @@ class AudioSettings(Enum):
     TRANSCODE = "transcode"
 
 
+class Simplicity(Enum):
+    SIMPLE = 0
+    ADVANCED = 1
+
+
+class SimpleVideoFormat(Enum):
+    MP4_H264 = "mp4_h264"
+    MP4_H265 = "mp4_h265"
+    WEBM = "webm"
+    GIF = "gif"
+
+
+def get_simple_format(
+    simple_video_format: SimpleVideoFormat, quality: int
+) -> tuple[VideoFormat, VideoEncoder, VideoPreset, int]:
+    container = None
+    encoder = None
+    video_preset = None
+    crf = None
+
+    container = {
```

Comment:
Those declarations shouldn't be necessary.

```suggestion
    container = {
```